app/javascript/articles/Article.jsx

```diff
@@ -99,7 +99,7 @@ export const Article = ({
         </a>
       </h4>
 
-      <TagList tags={article.tag_list || article.cached_tag_list_array} />
+      <TagList tags={article.tag_list} />
```

Comment:
@joshpuetz's work on the feed eliminates the need for checking for `cached_tag_list_array`. It is always just `tag_list` now.